jupyter-tensorflow-cuda-full/rockcraft.yaml

```diff
@@ -0,0 +1,251 @@
+      && echo "deb https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64 /" > /etc/apt/sources.list.d/nvidia-ml.list \
+      && apt-get -yq update \
+      && apt-get -yq install --no-install-recommends \
+      cm-super cuda-command-line-tools-${CUDA_VERSION/./-} cuda-nvrtc-${CUDA_VERSION/./-} \
+      libcublas-${CUDA_VERSION/./-} libcudnn8=${CUDNN_VERSION}+cuda${CUDA_VERSION} \
+      libcufft-${CUDA_VERSION/./-} libcurand-${CUDA_VERSION/./-} libcusolver-${CUDA_VERSION/./-} \
+      libcusparse-${CUDA_VERSION/./-} libfreetype6-dev libhdf5-serial-dev \
+      libnvinfer7=${LIBNVINFER_VERSION}+cuda${OLD_CUDA_VERSION} \
+      libnvinfer-plugin7=${LIBNVINFER_VERSION}+cuda${OLD_CUDA_VERSION} \
+      libzmq3-dev pkg-config \
+      # can't be used until NVIDIA updates (requires python < 3.7)
+      # python3-libnvinfer=${LIBNVINFER_VERSION}+cuda${CUDA_VERSION} \
+      && apt-get clean \
+      && rm -rf /var/lib/apt/lists/*
+
+      # tensorflow fix - wrong libcuda lib path (+ reconfigure dynamic linker run-time bindings)
+      ln -s ${CRAFT_PART_INSTALL}/usr/local/cuda/lib64/stubs/libcuda.so ${CRAFT_PART_INSTALL}/usr/local/cuda/lib64/stubs/libcuda.so.1 \
+      && echo "/usr/local/cuda/lib64/stubs" > /etc/ld.so.conf.d/z-cuda-stubs.conf \
+      && ldconfig
+
+      # copy CUDA config
+      mkdri -p ${CRAFT_PART_INSTALL}/etc/ld.so.conf.d/
+      cp -r /etc/ld.so.conf.d/z-cuda-stubs.conf ${CRAFT_PART_INSTALL}/etc/ld.so.conf.d/z-cuda-stubs.conf
+
+      # tensorflow fix - wrong libcusolver lib path
+      # https://github.com/tensorflow/tensorflow/issues/43947#issuecomment-748273679
+      ln -s ${CRAFT_PART_INSTALL}/usr/local/cuda-${CUDA_VERSION}/targets/x86_64-linux/lib/libcusolver.so.11 ${CRAFT_PART_INSTALL}/usr/local/cuda-${CUDA_VERSION}/targets/x86_64-linux/lib/libcusolver.so.10
+
+      # tensorflow fix - some tensorflow tools expect a `python` binary
+      ln -s $(which python3) ${CRAFT_PART_INSTALL}/usr/local/bin/python
+
+      # Install the jupyter-tensorflow CUDA requirements
+      cp ${CRAFT_PART_SRC}/components/example-notebook-servers/jupyter-tensorflow/cuda-requirements.txt cuda-requirements.txt
+      "${CONDA_BIN}/pip" install --no-cache-dir -r cuda-requirements.txt
+
+      # Install the jupyter-tensorflow-full requirements
+      cp ${CRAFT_PART_SRC}/components/example-notebook-servers/jupyter-tensorflow-full/requirements.txt requirements.txt
+      "${CONDA_BIN}/pip" install --no-cache-dir -r requirements.txt
+
+      # Create some directories for staging files
+      mkdir -p "${CRAFT_PART_INSTALL}/opt" "${CRAFT_PART_INSTALL}/etc" "${CRAFT_PART_INSTALL}/home"
+
+      # Write some config files for activating conda in user sessions
+      echo ". /opt/conda/etc/profile.d/conda.sh" >> "${HOME}/.bashrc"
+      echo ". /opt/conda/etc/profile.d/conda.sh" >> "${CRAFT_PART_INSTALL}/etc/profile"
```

Comment:
why a space ?

Reply:
Not related file got checked in with #35. Updated.